components/script/dom/servointernals.rs

```diff
@@ -73,7 +121,8 @@ impl ServoInternalsHelpers for ServoInternals {
             let in_realm_proof = AlreadyInRealm::assert_for_cx(cx);
             let global_scope = GlobalScope::from_context(*cx, InRealm::Already(&in_realm_proof));
             let url = global_scope.get_url();
-            url.scheme() == "about" && url.as_str() != "about:blank"
+            (url.scheme() == "about" && url.as_str() != "about:blank") ||
```

Comment:
Github doesn't allow adding review comments on non-diff lines, but the rustdoc of this function needs to be updated.